Test.C

```diff
@@ -150,14 +150,35 @@ int main(){
 	      while (encryptionPointer != null){
             // details about the encryption type itself is stored in encryptionStock
             // amount of this encryption type that the player has is stored in encryptionInventory
-            printf("%s\n", encryptionPointer->encryption->name);
-            encryptionPointer=encryptionPointer->next;
-        }  //current encrypt
+            printf("%s\n", encryptionPointer->encryption->name); //current encrypt
+            encryptionPointer=encryptionPointer->next; //pointer
+        } //end while loop
+		
         //iterate and print through current encryption
-
-        //receive credits NATHAN
-        //use equation to figure out how many credit user receives
+    }
+        
+    //receive credits
+		int sum;
+		int newCredits;
```

Comment:
```suggestion
		int sum = 0;
		int newCredits = 0;
```
Clears out garbage data